Formula/spdlog.rb

```diff
@@ -19,8 +19,6 @@ class Spdlog < Formula
   def install
     ENV.cxx11
 
-    inreplace "include/spdlog/tweakme.h", "// #define SPDLOG_FMT_EXTERNAL", "#define SPDLOG_FMT_EXTERNAL"
```

Comment:
Does this still use Homebrew `fmt` if we remove this? `tweakme.h` says this is what you need to do to use an external `fmt`.

Why not remove the `-DSPDLOG_FMT_EXTERNAL` from `spdlog.pc` instead?

Reply:
thanks for the feedback, I'll look into that.

Reply:
If `fmt` is external, then this definition should be present. Here's an example of what can happen if this isn't defined: https://github.com/gabime/spdlog/issues/904.

If a `#define` might already exist, then this should do the trick:
``` c++
#ifndef SPDLOG_FMT_EXTERNAL
#define SPDLOG_FMT_EXTERNAL
#endif
```

Maybe the Ruby replacement could drop that in?